5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -5,8 +5,14 @@
 # volume to override it.
 export MYSQL_DATADIR=/var/lib/mysql/data
 
+# Get prefix rather than hard-code it
+export MYSQL_PREFIX=$(which mysqld_safe|sed -e 's|/bin/mysqld_safe$||')
```

Comment:
is there a reason why we can't hard code the prefix that is worth installing 'which' ? ;-)